Code/GraphMol/AddHs.cpp

```diff
@@ -319,7 +319,8 @@ void setHydrogenCoords(ROMol *mol, unsigned int hydIdx, unsigned int heavyIdx) {
 }  // end of unnamed namespace
 
 namespace MolOps {
-void addHs(RWMol &mol, bool explicitOnly, bool addCoords) {
+void addHs(RWMol &mol, bool explicitOnly, bool addCoords,
+           const UINT_VECT *onlyOnAtoms) {
```

Comment:
Sometimes, but it's trickier to provide from Python and you can imagine situations where it's actually less efficient.

Using the UINT_VECT fits better with the way things like the fingerprinting code handles `fromAtoms` arguments.